### ISSUE TITLE
Move this repo to SARScene Organisation?

### ISSUE THREAD

**Issue by johnbrett** [ThinkingBIGRon/VolunteerTracker#2]
Hey guys,

How would you feel about moving this report to the sarscene organisation? https://github.com/sarscene @ThinkingBIGRon I think you have repo ownership currently?

I've forked it but can delete the fork if we move it. We're trying to aggregate all the projects in one place. 

Also - has everyone twitter handles or anything if they want to keep in touch? Mine is @_johnbrett

**Comment by evanepio** [comment]
I have no issues with moving the repo.

And I believe your twitter handle is @johnbrett_ :-)

Mine's @evanepio

**Comment by johnbrett** [comment]
You're correct, just changed it today, some service I wanted to use demanded a twitter handle as username but didn't allow usernames beginning with an underscrore :(

**Comment by ThinkingBIGRon** [comment]
I have no issue with moving the repo either.

When attempting a transfer, I received the following errors:  

```
SARscene/VolunteerTracker already exists and You don’t have admin rights to SARscene
```  

If you add me as an admin and remove the existing fork - I will attempt this again.  Glad to see these live on.

My Twitter handle is @ronkmyers.
